sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1280,6 +1324,112 @@ void ExtrinsicReflectorBasedCalibrator::calibrateSensors()
       "The calibrated poses differ considerably with the initial calibration. This may be either a "
       "fault of the algorithm or a bad calibration initialization");
   }
+
+  // for plotter and rviz display
+  output_calibration_distance_error = static_cast<float>(calibrated_2d_distance_error);
+  output_calibration_yaw_error = static_cast<float>(calibrated_2d_yaw_error);
+}
+
+void ExtrinsicReflectorBasedCalibrator::crossValEvaluation(
+  pcl::PointCloud<PointType>::Ptr lidar_points_pcs,
+  pcl::PointCloud<PointType>::Ptr radar_points_rcs)
+{
+  // Evaluate the cross validation metric using SVD
+  double cv_distance_error = 0.0;
+  double cv_yaw_error = 0.0;
+  auto get_yaw_error = [](const Eigen::Vector3d & v1, const Eigen::Vector3d & v2) -> double {
```

Comment:
Similar to another comment. Duplicated code

Reply:
lambda function `auto get_yaw_error` change to `double ExtrinsicReflectorBasedCalibrator::getYawError(const Eigen::Vector3d &v1, const Eigen::Vector3d &v2)`